gui/templates/system/bootenv_keep.html

```diff
@@ -0,0 +1,6 @@
+{% extends "freeadmin/generic_form.html" %}
+
+{% block form %}
+<tr><td colspan="2"><p translate="no">Do you want to keep bootenv?</p></td></tr>
```

Comment:
<p translate="no"> ? Makes no sense here

Please use:

{% trans "Do you want to set keep flag in this boot environment?" %}